packages/api/src/platforms/vtex/resolvers/validateCart.ts

```diff
@@ -366,13 +371,16 @@ export const validateCart = async (
     await clearOrderFormMessages(orderNumber, ctx)
   }
 
+  const sessionCookie = parse(ctx?.headers?.cookie ?? '')?.vtex_session
+  const jwt = parseJwt(sessionCookie)
```

Comment:
What do you think about renaming it to `sessionJwt`? (here and in the other occurrences).